src/Test/Utilities/Portable/TargetFrameworkUtil.cs

```diff
@@ -55,7 +56,7 @@ public enum TargetFramework
 
     public static class TargetFrameworkUtil
     {
-        public static MetadataReference StandardCSharpReference => CoreClrShim.IsRunningOnCoreClr ? NetStandard20.MicrosoftCSharpRef : TestBase.CSharpDesktopRef;
```

Comment:
>CoreClrShim.IsRunningOnCoreClr [](start = 67, length = 30)

It looks like you're replacing all uses of `CoreClrShim.IsRunningOnCoreClr`. Can that API be removed then?

Reply:
No. Scripting still uses it and pulling it out from there is a lot of work. It's on my list but wanted it to be a separate change as it's likely a bit of refactoring.